### PR TITLE
Update usage-scenario-shard-merge.md

### PR DESCRIPTION
**Pull request by ireneontheway**
### What is changed, added or deleted? (Required)

add scenario description

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/209
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: en/usage-scenario-shard-merge.md [hunk 1]
```diff
@@ -36,29 +36,29 @@ Assume that the upstream schemas are as follows:
...
-Assume that the downstream schema after replication is as follows:
+Assume that the downstream schema after migration is as follows:
 
 | Schema | Tables |
 |:------|:------|
 | user | information, log_north, log_east, log_south|
 | store | sale |
 
-## Replication solution
+## Migration solution
```
Comment (yikeke) [inline en/usage-scenario-shard-merge.md:59]: If you are updating "replication" to "migration", please also update "同步" to "迁移" in `zh/usage-scenario-shard-merge.md` to this PR. @ireneontheway

File: en/usage-scenario-shard-merge.md [hunk 2]
```diff
@@ -36,29 +36,29 @@ Assume that the upstream schemas are as follows:
     | store_01 | sale_01, sale_02 |
     | store_02 | sale_01, sale_02 |
 
-## Replication requirements
+## Migration requirements scenarios
```
Comment (yikeke) [inline en/usage-scenario-shard-merge.md:39]:
```suggestion
## Migration requirements
```

File: en/usage-scenario-shard-merge.md [hunk 3]
```diff
@@ -36,29 +36,29 @@ Assume that the upstream schemas are as follows:
...
-4. Filter out all the deletion operations in the `user`.`log_{north|south|east}` table of three upstream instances.
-5. Filter out all the deletion operations in the `user`.`information` table of three upstream instances.
-6. Filter out all the deletion operations in the `store_{01|02}`.`sale_{01|02}` table of three upstream instances.
-7. Filter out the `user`.`log_bak` table of three upstream instances.
-8. Because the `store_{01|02}`.`sale_{01|02}` tables have auto-increment primary keys of the `bigint` type, the conflict occurs when these tables are merged into TiDB. The following text will show you solutions to resolve and avoid the conflict.
+1. Merge tables with the same name. For example, merge the `user`.`information` table of three upstream instances to the downstream `user`.`information` table in TiDB.
+2. Merge tables with the different names. For example, merge the `user`.`log_{north|south|east}` table of three upstream instances to the downstream `user`.`log_{north|south|east}` table in TiDB.
+3. Merge sharded tables. For example, merge the `store_{01|02}`.`sale_{01|02}` table of three upstream instances to the downstream `store`.`sale` table in TiDB.
```
Comment (yikeke) [inline en/usage-scenario-shard-merge.md:43]:
```suggestion
1. Merge tables with the same name. For example, merge the `user`.`information` tables of three upstream instances to the downstream `user`.`information` table in TiDB.
2. Merge tables with different names. For example, merge the `user`.`log_{north|south|east}` tables of three upstream instances to the downstream `user`.`log_{north|south|east}` table in TiDB.
3. Merge sharded tables. For example, merge the `store_{01|02}`.`sale_{01|02}` tables of three upstream instances to the downstream `store`.`sale` table in TiDB.
```

File: en/usage-scenario-shard-merge.md [hunk 4]
```diff
@@ -36,29 +36,29 @@ Assume that the upstream schemas are as follows:
...
+4. Filter delete operations. For example, filter out all the deletion operations in the `user`.`log_{north|south|east}` table of three upstream instances.
+5. Filter delete operations. For example, filter out all the deletion operations in the `user`.`information` table of three upstream instances.
+6. Filter delete operations. For example, filter out all the deletion operations in the `store_{01|02}`.`sale_{01|02}` table of three upstream instances.
```
Comment (yikeke) [inline en/usage-scenario-shard-merge.md:46]:
```suggestion
4. Filter delete operations. For example, filter out all the delete operations in the `user`.`log_{north|south|east}` table of three upstream instances.
5. Filter delete operations. For example, filter out all the delete operations in the `user`.`information` table of three upstream instances.
6. Filter delete operations. For example, filter out all the delete operations in the `store_{01|02}`.`sale_{01|02}` table of three upstream instances.
```
To be consistent. Please also update other occurrences of this word in this doc. @ireneontheway

File: en/usage-scenario-shard-merge.md [hunk 5]
```diff
@@ -36,29 +36,29 @@ Assume that the upstream schemas are as follows:
...
+7. Use wildcards to filter specific tables. For example, filter out the `user`.`log_bak` table of three upstream instances using wildcard `user`.`log_*`.
```
Comment (yikeke) [inline en/usage-scenario-shard-merge.md:47]:
```suggestion
7. Use wildcards to filter specific tables. For example, filter out the `user`.`log_bak` tables of three upstream instances using wildcard `user`.`log_*`.
```

File: en/usage-scenario-shard-merge.md [hunk 6]
```diff
@@ -96,9 +96,9 @@ Assume that the downstream schema after replication is as follows:
 
     > **Note:**
     >
-    > The replication Requirements #4, #5 and #7 indicate that all the deletion operations in the `user` schema are filtered out, so a schema level filtering rule is configured here. However, the deletion operations of future tables in the `user` schema will also be filtered out.
+    > The migration Requirements #4 and #5 indicate that all the deletion operations in the `user` schema are filtered out, so a schema level filtering rule is configured here. And the deletion operations of tables participating in the future migration after the `user` schema will also be filtered out.
```
Comment (yikeke) [inline en/usage-scenario-shard-merge.md:99]:
```suggestion
    > The migration Requirements #4 and #5 indicate that all the deletion operations in the `user` schema are filtered out, so a schema level filtering rule is configured here. And the deletion operations of tables in the `user` schema participating in the future migration will also be filtered out.
```
deletion -> delete

File: en/usage-scenario-shard-merge.md (hunk 1 above)
Comment (ireneontheway) [reply en/usage-scenario-shard-merge.md:59]: Done

**Pull request merged**